### ISSUE TITLE
Port scalafix-testkit to utest

### ISSUE THREAD

**Issue by olafurpg** [scalacenter/scalafix#339]
If I remove all tokens from an input file to testkit, then I get the following error

```scala
[info] - scala/app/domain/entity.scala *** FAILED ***
[info]   java.util.NoSuchElementException: None.get
[info]   at scala.None$.get(Option.scala:349)
[info]   at scala.None$.get(Option.scala:347)
[info]   at scalafix.testkit.SemanticRuleSuite$.stripTestkitComments(SemanticRuleSuite.scala:20)
[info]   at scalafix.testkit.SemanticRuleSuite.$anonfun$runOn$1(SemanticRuleSuite.scala:123)
[info]   at scala.runtime.java8.JFunction0$mcZ$sp.apply(JFunction0$mcZ$sp.java:12)
```

Here 
https://github.com/scalacenter/scalafix/blob/c0a26983514599172b30e515c9b2029cd92e1f64/scalafix-testkit/src/main/scala/scalafix/testkit/SemanticRuleSuite.scala#L23
we can return a bogus token `.getOrElse("".tokenize.get(0))` instead of calling `.get`

**Comment by olafurpg** [comment]
I think a better solution for this issue would be to port scalafix-testkit to utest that supports testOnly for dynamically created tests, then we can remove the ONLY logic!

**Comment by olafurpg** [comment]
This is implemented in #411 but I closed the PR to limit breaking changes. We can revive that PR when we start planning for v0.6

**Comment by olafurpg** [comment]
Let's stick with ScalaTest since changing is a breaking change with questionable value.
